lncfg/protocol.go

```diff
@@ -12,6 +12,10 @@ type ProtocolOptions struct {
 	// Anchors should be set if we want to support opening or accepting
 	// channels having the anchor commitment type.
 	Anchors bool `long:"anchors" description:"EXPERIMENTAL: enable experimental support for anchor commitments, won't work with watchtowers"`
+
+	// WumboChans should be set if we want to disable support for wumbo
+	// channels, which is the opposite of mini.
+	WumboChans bool `long:"wumbo" description:"if set, then lnd will create and accept requests for wumbo channels"`
```

Comment:
name is still ambiguous, `large-channels` is imo more descriptive esp for an international audience. imo help message should also say specifically that this lets the user create channels larger than the original .16 BTC capacity limit rather than assuming they know what wumbo channels are.

Reply:
The tacked on channels. The description itself is adequate IMO, now that I've tacked on that it's greater than 0.16 BTC. Those that learn of the new feature will likely learn of it via the release notes, or our blog post which will explain the feature. Most people in the existing community know what we mean by wumbo. Also since it's opt-in, those that seek to enable it already know what their need is. Those that try to make channels larger w/o this option will then seek to find out why they can't and discover the option.

Reply:
I still don't it that approachable to new users a few years down the line who will have missed the wumbo saga, even if people today will know and understand, but i think i've made my case.